.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 160
```

Comment:
160 is a pretty long line length, we've been using 120 across most repos

Reply:
Ideally we want it standardised throughout sdc, as it's different in eq, ras, sdx, open for discussion?